relevanceai/vector_tools/cluster.py

```diff
@@ -55,6 +55,7 @@ def fit_documents(
 
         """
         if len(vector_field) == 1:
+            docs = [doc for doc in docs if vector_field[0] in doc.keys()]
```

Comment:
This will fail if the vector field is nested. 
Use this to help you work with nested fields: https://github.com/RelevanceAI/python-doc-utils/blob/main/doc_utils/read_utils.py#L136